jib-core/src/main/java/com/google/cloud/tools/jib/frontend/ExposedPortsParser.java

```diff
@@ -80,14 +79,12 @@
 
       // Warn for possibly invalid port numbers
       if (min < 1 || max > 65535) {
-        // TODO: Add details/use HelpfulSuggestions for these warnings
-        buildLogger.warn("Port number '" + port + "' is out of usual range (1-65535).");
+        throw new NumberFormatException(
```

Comment:
I wonder if NumberFormatException is not a great exception to throw here. If it's a user configuration error, maybe we should expose it to whoever needs to handle it?